mage_ai/cli/main.py

```diff
@@ -36,6 +40,18 @@ def list_commands(self, ctx: Context):
 START_DBT_DOCS_INSTANCE_DEFAULT = typer.Option('0', help='')
 START_INSTANCE_TYPE_DEFAULT = typer.Option(
     InstanceType.SERVER_AND_SCHEDULER.value, help='specify the instance type.')
+START_PROJECT_TYPE_DEFAULT = typer.Option(
+    'standalone',
+    help='create project of this type if does not exist, options are main, sub, or standalone',
+)
+START_CLUSTER_TYPE_DEFAULT = typer.Option(
```

Comment:
Why is the default cluster type "k8s"? Should it be None by default?

Reply:
yeah you're right